react/components/ExtensionPoint/ComponentLoader.tsx

```diff
@@ -76,55 +26,12 @@ const ComponentLoader: FunctionComponent<Props> = props => {
    * computed once during runtime, since we know that, even if componentProps
    * is updated, the props that function as Slots will NOT change.
    */
-  const slots = useMemo(() => {
```

Comment:
~I guess you can move the comment from above to the `componentGetter` too :)~
This can be removed since you've already moved it to `useSlots`